add-on/src/popup/browser-action/store.js

```diff
@@ -27,7 +27,7 @@ export default (state, emitter) => {
     publicSubdomainGatewayUrl: null,
     gatewayAddress: null,
     swarmPeers: null,
-    gatewayVersion: null,
+    kuboRpcBackendVersion: null,
```

Comment:
:information_source:  renamed to make it clear this is version of RPC backend, and not gateway.
(if we want versioned gateways, we should expose gateway-conformance version, but this is future work)